libraries/plugins/peerplays_sidechain/sidechain_net_handler_bitcoin.cpp

```diff
@@ -480,13 +483,13 @@ std::vector<btc_txout> bitcoin_rpc_client::listunspent() {
 std::vector<btc_txout> bitcoin_rpc_client::listunspent_by_address_and_amount(const std::string &address, double minimum_amount) {
    std::string body = std::string("{\"jsonrpc\": \"1.0\", \"id\":\"pp_plugin\", \"method\": "
                                   "\"listunspent\", \"params\": [");
-   body += std::string("1,999999,[\"");
+   body += std::string("0,999999,[\"");
```

Comment:
We cant have 0 here. We actually need like 5 or 6. Transactions with 0 confirmations might not exists when the block is mined.